README.md

```diff
@@ -82,42 +98,53 @@ app.use('/mbaas', mbaasExpress.mbaas);
 app.use(mbaasExpress.fhmiddleware());
 
 // Module used to create RESTful router instances
-var fhRestExpressRouter = require('fh-rest-express-router');
+const fhRestExpressRouter = require('fh-rest-express-router');
 
 // Module that RESTful router will use to retrieve data
 // Note: this is not yet developed
-var fhRestMemoryAdapter = require('fh-rest-memory-adapter');
+const fhRestMemoryAdapter = require('fh-rest-memory-adapter');
 
 // Creates a handler for incoming HTTP requests that want to perform CRUDL
 // operations on the "orders" table in your MySQL database
-var ordersRouter = fhRestExpressRouter({
+const ordersRouter = fhRestExpressRouter({
   // The name of this router
   name: 'orders',
 
   // Joi schemas that validate the querystring/body is safe to pass to
   // an adapter instance. Only for list, create, and update
-  validations: {
-    'list': [require('./validate-list')],
-    'create': [require('./validate-create')],
-    'update': [require('./validate-update')]
+  bodyAndQueryValidations: {
+    'list': [{
+      schema: require('./validate-list-schema.js'),
+      options: {
+        allowUnknown: true //Allow parameters not in Joi schema through
+      }
+    }],
+    'create': [{
+      schema: require('./validate-create-schema.js'),
+      options: {
+        allowUnknown: true // Remove parameters not in Joi schema
```

Comment:
`allowUnknown: true // Remove parameters not in Joi schema`
should be
`stripUnknown: true // Remove parameters not in Joi schema`